tools/appslib/logging_sender.py

```diff
@@ -1,16 +1,37 @@
 #!/usr/bin/env python3
 
 import subprocess
-from shutil import which
 import logging
 import logging.handlers
 
 
-def send_to_matrix(message: str) -> None:
-    if which("sendxmpppy") is None:
-        logging.warning("Could not send error via xmpp.")
-        return
-    subprocess.call(["sendxmpppy", message], stdout=subprocess.DEVNULL)
+def notify(message, channel):
```

Comment:
I would prefer keeping the more verbose function name send_to_matrix

Reply:
not me, I prefer not to name a protocol to be able to notify on several if needed later

Reply:
Alright, fine with me.